web/src/frontend/src/nav-soknad/faktum/SporsmalFaktum.tsx

```diff
@@ -19,11 +20,26 @@ export interface OwnProps {
 type Props = OwnProps & InjectedFaktumComponentProps & InjectedIntlProps;
 
 class SporsmalFaktum extends React.Component<Props, {}> {
+	constructor(props: Props) {
+		super(props);
+		this.handleOnBlur = this.handleOnBlur.bind(this);
+	}
+
+	handleOnBlur(evt: any) {
+		if (this.props.renderValideringsfeil) {
+			setTimeout(() => {
```

Comment:
Hva gjør det her?

Reply:
Når en får blur fra et spørsmål så sjekker en om aktivt element ligger utenfor spørsmålet. Dersom det er tilfellet, revalideres faktumet. Timeout må til for at nettleser skal få satt riktig document.activeElement